### DISCUSSION TITLE
Folder view side by side

### DISCUSSION THREAD

**Discussion by dvbhardware** [WinMerge/winmerge#1096] [General]
I cannot find a setting to show contents of folders side by side
Make no sense to  me to compare in a single column and highlight the different files

![folder view](https://user-images.githubusercontent.com/70298488/146793696-c33320b8-68ae-4abc-9a1f-bb1ad9dfabb4.jpg)
.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge doesn't have that feature. 

related discussion https://github.com/WinMerge/winmerge/discussions/868
